cmds/core/yes/yes_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2017-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bufio"
+	"testing"
+
+	"github.com/u-root/u-root/pkg/testutil"
+)
+
+func TestMain(m *testing.M) {
+	testutil.Run(m, main)
+}
+
+func TestYes(t *testing.T) {
+	type test struct {
+		name        string
+		in          []string
+		expected    string
+		closeOrKill bool
+	}
+	tests := []test{
+		{name: "noParameterCloseTest", in: []string{}, expected: "y", closeOrKill: true},
+		{name: "noParameterKillTest", in: []string{}, expected: "y", closeOrKill: false},
+		{name: "oneParameterCloseTest", in: []string{"hi"}, expected: "hi", closeOrKill: true},
+		{name: "oneParameterKillTest", in: []string{"hi"}, expected: "hi", closeOrKill: false},
+		{name: "fourParameterCloseTest", in: []string{"hi", "how", "are", "you"}, expected: "hi how are you", closeOrKill: true},
+		{name: "fourParameterKillTest", in: []string{"hi", "how", "are", "you"}, expected: "hi how are you", closeOrKill: false},
+	}
+	for _, v := range tests {
+		t.Run(v.name, func(t *testing.T) {
+			cmd := testutil.Command(t, v.in...)
+			stdout, err := cmd.StdoutPipe()
+			if err != nil {
+				t.Fatalf("Failed to get stdout of yes command: got %v, want nil", err)
+			}
+			if err := cmd.Start(); err != nil {
+				t.Fatalf("Failed to start yes command: got %v, want nil", err)
+			}
+			in := bufio.NewScanner(stdout)
+			for i := 0; i < 1000; i++ {
+				if !in.Scan() {
+					t.Fatalf("Could not scan: got %v, want nil", in.Err())
+				}
+				if text := in.Text(); text != v.expected {
+					t.Errorf("Got %v at iteration %d, want %v", text, i, v.expected)
+					break
+				}
+			}
+			if v.closeOrKill == true {
```

Comment:
so one last thing, the name closeOrKill confused me. Is it true for close, or true for kill. I would recommend 
closeStdout